ts/features/bonus/common/components/BonusInformationComponent.tsx

```diff
@@ -9,7 +9,7 @@ import {
 import * as AR from "fp-ts/lib/Array";
 import { constNull, pipe } from "fp-ts/lib/function";
 import * as O from "fp-ts/lib/Option";
-import { ComponentProps, useContext } from "react";
+import React, { ComponentProps, useContext, useImperativeHandle } from "react";
```

Comment:
You don’t need to import the default React object, instead, you can import `forwardRef` directly along with the hooks and types you need:

```suggestion
import {
  ComponentProps,
  forwardRef,
  useContext,
  useImperativeHandle
} from "react";
```

Reply:
Addessed here: 7ee27d7ca7bb1359ba0c0e01865cc1fcce69c00d